src/OutputFormatter/SarbOutputFormatter.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SensioLabs\Deptrac\OutputFormatter;
+
+use SensioLabs\Deptrac\DependencyContext;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class SarbOutputFormatter implements OutputFormatterInterface
+{
+    private static $argument_dump_sarb = 'dump-sarb';
+
+    public function getName(): string
+    {
+        return 'sarb';
+    }
+
+    /**
+     * @return OutputFormatterOption[]
+     */
+    public function configureOptions(): array
+    {
+        return [
+            OutputFormatterOption::newValueOption(static::$argument_dump_sarb, 'path to a dumped sarb file', './sarb.json'),
+        ];
+    }
+
+    public function enabledByDefault(): bool
+    {
+        return false;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
+    public function finish(
+        DependencyContext $dependencyContext,
+        OutputInterface $output,
+        OutputFormatterInput $outputFormatterInput
+    ): void {
+
+        $sarbIssues = [];
+
+        foreach ($dependencyContext->getViolations() as $violation) {
+
+            $dependency = $violation->getDependency();
```

Comment:
We need to check if the violation has been skipped or waiting for #266

Reply:
Not sure I fully understand... 

Where I have [this code](https://github.com/DaveLiddament/deptrac/blob/feature/sarb-format-output/src/OutputFormatter/SarbOutputFormatter.php#L47):
```
        foreach ($dependencyContext->getViolations() as $violation) {
            $dependency = $violation->getDependency();
           ... add violation to outputted values ....
      }
```

Should I have something like this :

```
        $ignoredViolations = $dependencyContext->getIgnoredViolations();

        foreach ($dependencyContext->getViolations() as $violation) {

            if ($violation is one of the $ignoredViolations) {
                continue;
           }

            $dependency = $violation->getDependency();
           ... add violation to outputted values ....
      }
```

Reply:
Sorry, there is a method `isViolationSkipped(...)` on `DependencContext` -> https://github.com/sensiolabs-de/deptrac/blob/master/src/DependencyContext.php#L102